http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -1746,17 +1743,6 @@ case class AppCannotBeDeletedException(cloudContext: CloudContext,
       extraMessageInLogging = extraMsg
     )
 
-case class AppCannotBeDeletedByWorkspaceIdException(workspaceId: WorkspaceId,
```

Comment:
was this just never used? (same question with the other removed exception below)

Reply:
yeah, this is just removing code intellIJ told me is unused